src/main/scala/org/allenai/ike/patterns/PatternUtilities.scala

```diff
@@ -0,0 +1,58 @@
+package org.allenai.ike.patterns
+
+import com.typesafe.config.ConfigFactory
+import org.allenai.ike.{SearchConfig, SearchRequest}
+import scala.collection.JavaConverters._
+
+/**
+  * Created by matt on 8/31/16.
+  */
+
+object PatternUtilities {
+  /** Given a Seq of NamedPattern, create a Map where the key is the pattern's name, and the
+    * value is a SearchRequest.
+    *
+    * In this manner you can access a searchrequest by its name
+    *
+    * @param namedPatterns A Seq[NamedPattern]
+    * @return a Map[String, SearchRequest] where the key is the name from a NamedPattern
+    */
+  def createSearchers(namedPatterns: Seq[NamedPattern]): Map[String, SearchRequest] = {
+    val hugeLimit = 100000000
+    namedPatterns.map{ case namedPattern =>
+      (namedPattern.name, SearchRequest(Left(namedPattern.pattern), None, None, SearchConfig(hugeLimit, 1)))
+    }.toMap
+  }
+
+  /** Takes a config file and creates the NamedPattern extractions patterns
+    *
+    * The config file with the patterns looks like:
```

Comment:
This part of the description can be covered by the test config now.  I'd remove it here to avoid duplication of code; if we teak the format this will drift away from correct.